src/components/dialog-full-screen/dialog-full-screen.stories.js

```diff
@@ -58,7 +66,37 @@ storiesOf('Dialog Full Screen', module)
         </State>
       </div>
     );
-  }, {
-    notes: { markdown: notes },
-    knobs: { escapeHTML: false }
-  });
+  }, storyParameters)
+  .add('with sticky footer', () => {
+    const title = text('title', 'Example Dialog');
+    const subtitle = text('subtitle', 'Example Subtitle');
+    const children = text('children', 'Text Content');
+    const enableBackgroundUI = boolean('enableBackgroundUI', false);
+    const disableEscKey = boolean('disableEscKey', false);
+    const ariaRole = text('ariaRole', 'dialog');
+    const stickyFooter = boolean('stickyFooter', true);
```

Comment:
If you have a story for specifically showing a sticky footer, I would suggest you don't need a knob for it.

Reply:
OK I've removed it. Thanks.